cache-core/src/main/java/io/micronaut/cache/annotation/InvalidateOperations.java

```diff
@@ -32,6 +32,7 @@
 @Retention(RetentionPolicy.RUNTIME)
 @Inherited
 @Documented
+@CacheAnnotation
```

Comment:
Why was this added here? If `@CacheConfig` was not already there then it shouldn't need to be here either

Reply:
Actually, it is required. When removed, the TCK fails for tests that hit methods like:

```groovy
@PutOperations([
        @CachePut('counter'),
        @CachePut('counter2')

])
int increment2(String name) {
    int value = counters2.computeIfAbsent(name, { 0 })
    counters2.put(name, ++value)
    return value
}
```